apps/node-fastify/src/app.ts

```diff
@@ -46,6 +48,16 @@ export async function createApp(opts: FastifyServerOptions = {}): Promise<Fastif
 
   app.decorate('orbSync', orbSync);
 
+  if (config.CRON_REFRESH_STALE_SUBSCRIPTIONS) {
+    nodeCron.schedule('*/15 * * * *', async () => {
```

Comment:
Is this once an hour? A comment would help here

Reply:
It's on every 15th minute. Will add a comment for whatever the final schedule will be